libraries/Microsoft.Bot.Builder.Dialogs.Declarative/ExpressionProperty.cs

```diff
@@ -40,7 +40,7 @@ public ExpressionProperty(T value)
         /// </summary>
         public string Expression
         {
-            get { return expression.ToString(); }
+            get { return expression?.ToString(); }
             set { expression = new ExpressionEngine().Parse(value); }
```

Comment:
I know this isn't in your change, but how does this work?  This produces a value of type Expression, not string.

Reply:
yep